cmd/jujud/agent/upgrade_test.go

```diff
@@ -806,7 +806,18 @@ func (s *UpgradeSuite) assertCommonUpgrades(c *gc.C) {
 	c.Assert(cmds, gc.HasLen, 1)
 	args := cmds[0].Args
 	c.Assert(len(args), jc.GreaterThan, 1)
-	c.Assert(args[0], gc.Equals, "apt-get")
+
+	os, err := version.GetOSFromSeries(version.Current.Series)
+	c.Assert(err, gc.IsNil)
```

Comment:
There's a relatively new(few months old I think) function in juju testing. 

You would do: c.Assert(err, jc.ErrorIsNil).

If jc(juju checkers) is not imported it's usually:
jc "github.com/juju/testing/checkers"